docs/source/installation.rst

```diff
@@ -100,98 +100,270 @@ Step 1: Setup dedicated python environment
 
-                cd ivadomed
+                        .. tab:: Windows
 
-                pip install -e .
+                            with CUDA 10.2:
 
+                            ::
+                                
+                                pip install ivadomed --extra-index-url https://download.pytorch.org/whl/cu102
 
-Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                            and, with CUDA 11.1:
 
-    .. tabs::
+                            ::
 
-        .. group-tab:: PyPI Installation
+                                pip install ivadomed --extra-index-url https://download.pytorch.org/whl/cu111
 
-            .. tabs::
 
-                .. tab:: Nvidia GPU Support
+                .. tab:: Source Installation
+
+                    Bleeding-edge developments are available on the master branch of the project
+                    on Github. To install ``ivadomed`` from source
+
+                    .. tabs::
+
+                        .. tab:: Linux
+
+                            with CUDA 10.2:
+                            
+                            ::
+                   
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e .
+
+                            and, with CUDA 11.1:
+
+                            ::
+
+                                git clone https://github.com/ivadomed/ivadomed.git
+
+                                cd ivadomed
+
+                                pip install -e . --extra-index-url https://download.pytorch.org/whl/cu111
+
+                        .. tab:: Windows 
+
+                            with CUDA 10.2:
+
+                            ::
 
-                    ``ivadomed`` requires CUDA11 to execute properly. If you have a nvidia GPU, try to look up its Cuda Compute Score `here <https://developer.nvidia.com/cuda-gpus>`__, which needs to be > 3.5 to support CUDA11. Then, make sure to upgrade to nvidia driver to be at least v450+ or newer.
+                                git clone https://github.com/ivadomed/ivadomed.git
 
-                    You can use ``nvidia-smi`` in both Linux and Windows to check for driver CUDA Version listed at the top right of the output console. On Linux, simply type in ``nvidia-smi`` in any console to see the output. On windows, you will need to locate the `nvidia-smi.exe` tool by following the instructions on `this page <https://stackoverflow.com/a/57100016>`__.
+                                cd ivadomed
 
-                    If you have a compatible NVIDIA GPU that supports CUDA11, and you have a recent enough driver installed, then run the following command:
+                                pip install -e . --extra-index-url https://download.pytorch.org/whl/cu102
```

Comment:
My suggestion above drops Windows support entirely, because @jcohenadad [said we should](https://github.com/neuropoly/computers/issues/202#issuecomment-1139710695), but it would be good if the trick to getting GPUs working on Windows was saved _somewhere_.

I'm not sure where though. Maybe it could go in CONTRIBUTING.md along with the rest of this section, where we aren't as constrained by having to write for a wider audience and can dump reams of technical details with less care?

Or maybe we could tuck it into the GPU section in a `tabs:::` afterall and just ignore @jcohenadad's ask -- I wrote in my suggestion "Not officially supported nor tested", but at the moment GPU support on Linux isn't tested either, except by our own research, so maybe it's not so much of a stretch to just tuck in the Windows case too and get around to testing it [mañana](https://www.wordnik.com/words/ma%C3%B1ana).

Reply:
GPU testing will resume outside of the existing setup for sanity check reasons. It will be a completely independent and private process though as our existing GitHub workflow is not flexible enough to accommodate it and cannot securely make it integrate with GitHub in its current implementation.  I will open bugs on issue found manually for now.